src/Entity/Customer.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace App\Entity;
+
+use App\Model\Message;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity
+ */
+class Customer
+{
+
+    /**
+     *
+     * @ORM\Column(name="`customer_code`", type="string", length=32, nullable=false)
+     *
+     * @var string
+     */
+    public string $code;
```

Comment:
This can be an UID / ID of entity , but it is missing certain ORM comment which indicates that.